src/query/service/tests/it/storages/fuse/pruning.rs

```diff
@@ -78,8 +78,8 @@ async fn test_block_pruner() -> Result<()> {
         table: test_tbl_name.to_string(),
         schema: test_schema.clone(),
         engine: Engine::Fuse,
-        storage_params: ctx.get_storage_params(),
-        path: ctx.get_storage_operator()?.metadata().root().to_string(),
+        storage_params: None,
```

Comment:
`storage_params` and `path` should be some or none together. So why not make them inside a new struct ?

Reply:
The `path` can be gotten from the operator, so I think we no need to store it.